partner_communication_switzerland/migrations/12.0.1.1.4/post-migration.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, installed_version):
+    if not installed_version:
+        return
+
+    # Add activities "for dont inform" sponsors
+    sponsorships = env["res.partner"]\
+        .search([('global_communication_delivery_preference', '=', 'none')])\
+        .sponsorship_ids\
+        .filtered(lambda it: it.state in ["waiting", 'active'])
```

Comment:
I would rather only use a search query. We also want to exclude a special partner that has more than 500 sponsorships.
```suggestion
    sponsorships = env["recurring.contract"].search([
        ('correspondent_id.global_communication_delivery_preference', '=', 'none'),
        ('state', 'in', ['waiting','active']),
        ('correspondent_id.ref', '!=', '1502623')])
```

Reply:
Alright, and damn for the special partner!